frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -110,10 +101,8 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
   const featureGroupRef = useRef<any>();
   const draftFeatureGroupRef = useRef<any>();
   const filterState = useFilterContext();
-  const dispatch = useDispatch();
```

Comment:
don't need this anymore as the selectedProperty marker no longer has popup functionality to load data.